### ISSUE TITLE
FR: Add an option to hide available content from "Trending" and "Because you watched"

### ISSUE THREAD

**Issue by hbooo** [petio-team/petio#304]
**Is your feature request related to a problem? Please describe.**
If you have a lot of available content in your plex, "Trending" can be useless because you already own first content trending.

**Describe the solution you'd like**
Add an option to hide available content from "Trending".
It also could be interesting to have an option to hide already seen content from "Because you watch". 

Before : 
![image](https://user-images.githubusercontent.com/242125/112317761-a971c080-8cac-11eb-8fe3-30943fbc26e3.png)

After : 
![image](https://user-images.githubusercontent.com/242125/112317940-d02ff700-8cac-11eb-8b75-657467850d16.png)

And for Because you watched
![image](https://user-images.githubusercontent.com/242125/112332470-0d4eb600-8cba-11eb-9ef2-7bbe785b356c.png)

After
![image](https://user-images.githubusercontent.com/242125/112332620-2eafa200-8cba-11eb-8d32-21330f4cf7e3.png)
The Good Place is still displayed because I didn't watch it yet

**Comment by AshDyson** [comment]
Ok so a few FRs in one, 

Because you watched does exclude your own history, just the history is only the past 500 items, if its outside of the history scope then we just show it. We can extend the history scope but it can cause slower parsing on slower machines so it seemed like this was long enough to be useful but not too long as to slow things down. Also note: an episode play counts as one history item so loads of tv can fill it up pretty fast.

Trending is just whatevers popular on tmdb, we use it as the default view when you haven't searched for anything yet so Trending is a required element. Also trending changes daily so you'll get new results as they start to trend.

We do however in the upcoming patch have an option to toggle "popular on plex" as this data is completely driven by whats already on plex.

Tl;dr - Trending can't be turned off, we already do hide _recently played_ movies / tv

**Comment by hbooo** [comment]
Hi, 

I wasn't asking to remove entirely trending but only already owned content. I agree that trending is central and that's why I believe it could be useful to have accurate trending movies/shows in order to discover some new items.

**Comment by AshDyson** [comment]
> Hi, 
> 
> 
> 
> I wasn't asking to remove entirely trending but only already owned content. I agree that trending is central and that's why I believe it could be useful to have accurate trending movies/shows in order to discover some new items.

Fir everything else I'm totally fine to have it filterable but because trending is like the default it needs to load fast to populate the default search view which is the first thing people hit. If we filter out on server stuff this causes issues if we have to load in 3 pages of results before you hit stuff not on your server which will cause a significant slow down of the apps "ready state". So yeah anything else is fine but trending we need to be static for performance. Hope this makes sense!
